spec/controllers/queue_items_controller_spec.rb

```diff
@@ -124,7 +124,7 @@
       end
       
       it "does not change the queue items" do
-        expect(current_user.queue_items).to eq([queue_item1, queue_item2])
+        expect(current_user.queue_items).to match_array([queue_item1, queue_item2])
```

Comment:
`match_array` does not verify order you should use `eq` to verify the order didn't change.